### PR TITLE
CONTENT ADD: Add front end course

### PR DESCRIPTION
**Pull request by Soedipo**
### I add front end course, actually I just edit it from .txt format before into markdown.
**Here is the list:**
1. setup
2. html_basic
3. html_body_content_pt1
4. html_body_content_pt2
5. html_section
6. css_setup
7. css_selector
8. css_properties
9. css_color
10. css_background
11. css_front
12. css_box_model_unit
13. css_padding
14. css_border
15. css_margin
16. css_display
17. css_flex
18. css_web_nav_1
19. css_web_nav_2
20. css_web_content_1
21. css_web_content_2
22. css_web_responsive
23. js_setup
24. js_dom
25. js_dom_manipulation
26. references

### CODE REVIEW

**Comment by fakhrip** [comment]
I will wait for @adhiiimf approval on this one, but just to add a bit, i think adding some more explanation to the js dom manipulation and other things in the js stuff might help more for the mentee 👌

And also you might want to remove this one now, idk if its still relevant in this case or not 😅

https://github.com/Soedipo/2023-Academy/blob/main/resources/front_end/26_references.md#L1

**Review by adhiiimf — CHANGES_REQUESTED** [review]
I'm Sorry, but first please rename all of this filename to be like this, 001_filename, 002_filename, etc. Because I think it'll be facilitate the academician to learning this path from 1 course 'til final course of this frontend path. Thanks. @Soedipo 
image

**Comment by adhiiimf** [comment]
> I will wait for @adhiiimf approval on this one, but just to add a bit, i think adding some more explanation to the js dom manipulation and other things in the js stuff might help more for the mentee 👌
> 
> And also you might want to remove this one now, idk if its still relevant in this case or not 😅
> 
> https://github.com/Soedipo/2023-Academy/blob/main/resources/front_end/26_references.md#L1

I Think this is good idea from @fakhrip, adding some `simple` explanation is better than you give so many references. coz I believe with more explanation and example of that will be more efficient. But This is up to u @Soedipo 😄

File: resources/front_end/002_html_basic.md
Comment (adhiiimf) [inline resources/front_end/002_html_basic.md]: Better if u explain this. Coz I think not everyone know about this

File: resources/front_end/019_css_web_nav_pt2.md
Comment (adhiiimf) [inline resources/front_end/019_css_web_nav_pt2.md]: have u explain why u using double nested selector in this case?

File: resources/front_end/019_css_web_nav_pt2.md
Comment (adhiiimf) [inline resources/front_end/019_css_web_nav_pt2.md]: this is too

File: resources/front_end/019_css_web_nav_pt2.md
Comment (adhiiimf) [inline resources/front_end/019_css_web_nav_pt2.md]: yap this is too

File: resources/front_end/022_css_web_responsive.md
Comment (adhiiimf) [inline resources/front_end/022_css_web_responsive.md]: Give case min-width will be better

File: resources/front_end/023_js_setup.md
Comment (adhiiimf) [inline resources/front_end/023_js_setup.md]: Will be better if u explain why u using id than class or others selector. Or just add explain like this => id is a unique identifier (based on rules) html

**Review by adhiiimf — CHANGES_REQUESTED** [review]
Okeyy overall good. But I think you miss some essential component in html like button, what is meta, link, title, etc

**Review by adhiiimf — APPROVED** [review]
Nice Dip, enough from me.

**Pull request merged**
